app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -109,7 +112,9 @@ class HomePresenter(
 
     init {
         val userSeedPhrase = seedPhraseProvider.getUserSeedPhrase().seedPhrase
-        ethereumInteractor.setup(userSeedPhrase = userSeedPhrase)
+        if (userSeedPhrase.isNotEmpty()) {
```

Comment:
please put else branch here and log some Throwable for us to know how many users don't have user seed phrase like this: 
if (userSeedPhrase.isNotEmpty(){
...
} else {
Timber.e(NoSeedPhraseForEth(), "ETH is not init, no seed phrase")
}